### PR TITLE
setup rails and create a model

### PR DESCRIPTION
**Pull request by huzaifakhan0308**
- setup rails project
- create a data model

### CODE REVIEW

File: db/migrate/20230503102449_schema.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Schema < ActiveRecord::Migration[7.0]
+  def change
+  end
+end
```
Comment (emmanuellekamwa) [inline db/migrate/20230503102449_schema.rb]: - [ ] You are supposed to create a migration file for creating user table, one for creating posts table, one for creating comments table and one for creating likes table but I can't see it in your project. Please just add that to your project.

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUESTED ❌⛔🛑**
Hello @huzaifakhan0308 👋
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Good commit message.

## Required Changes ♻️
_Check the comments under the review._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20230503123427_create_users.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :user do |t|
+      t.text :name
+      t.text :photo
+      t.text :bio
+      t.date :createdat
+      t.date :updatedat
```
Comment (Kamba56) [inline db/migrate/20230503123427_create_users.rb:8]: - [optional] The createdat and updatedat columns are automatically created by rails  using the `t.timestamps`. Kindly remove these, please.

File: db/migrate/20230503123537_create_comments.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.date :createdat
+      t.date :updatedat
```
Comment (Kamba56) [inline db/migrate/20230503123537_create_comments.rb:6]: - [optional] The createdat and updatedat columns are automatically created by rails  using the `t.timestamps`. Kindly remove these please.

File: db/migrate/20230503123549_create_likes.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.integer :author_id
+      t.integer :post_id
+      t.date :createdat
+      t.date :updatedat
```
Comment (Kamba56) [inline db/migrate/20230503123549_create_likes.rb:7]: - [optional] The createdat and updatedat columns are automatically created by rails  using the `t.timestamps`. Kindly remove these please.

File: db/migrate/20230503123612_create_posts.rb [hunk 5]
```diff
@@ -0,0 +1,15 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.integer :author_id
+      t.text :title
+      t.text :text
+      t.date :createdat
+      t.date :updatedat
```
Comment (Kamba56) [inline db/migrate/20230503123612_create_posts.rb:8]: - [optional] The createdat and updatedat columns are automatically created by rails using the `t.timestamps`. Kindly remove these please.

**Review by Kamba56 — APPROVED** [review]
Hi @huzaifakhan0308, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
## Highlights
- [x] GOod work using migrations to create tables.!👏👏👏

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**